src/i18n/messages/es.json

```diff
@@ -1,4 +1,5 @@
 {
   "profile.no.certificates": "Aún no tienes ningún certificado.",
-  "profile.bio.empty": "Añada una breve biografía"
+  "profile.bio.about.me": "Sobre Mi",
+  "profile.bio.empty": "Añade una breve biografía"
```

Comment:
Do you plan on leaving this file in?  Probably fine either way.  Just a reminder to remove later if you planned on removing it.

Reply:
This is the file that drives Spanish, so yes.  :-)